src/__experimental__/components/search/search.style.js

```diff
@@ -1,17 +1,31 @@
 import styled, { css } from "styled-components";
 import StyledInputPresentation from "../input/input-presentation.style";
+import StyledInput from "../input/input.style";
 import StyledIcon from "../../../components/icon/icon.style";
 import StyledButton from "../../../components/button/button.style";
 import { baseTheme } from "../../../style/themes";
 import StyledFormField from "../form-field/form-field.style";
 
 const StyledSearch = styled.div`
-  ${({ isFocused, searchWidth, searchIsActive, searchHasValue, theme }) => {
+  ${({
+    isFocused,
+    searchWidth,
+    searchIsActive,
+    searchHasValue,
+    searchButton,
+    theme,
+    variant,
+  }) => {
+    const darkTheme = variant === "dark";
     return css`
       width: ${searchWidth ? `${searchWidth}` : "100%"};
       padding-bottom: 2px;
       background-color: transparent;
-      border-bottom: 2px solid ${theme.search.passive};
+      border-bottom: 2px solid ${
+        darkTheme
+          ? `${theme.search.darkThemeBorder}`
+          : `${theme.search.passive}`
```

Comment:
you could make it a const, e.g.:
```
const variantColor = darkTheme
      ? `${theme.search.darkThemeBorder}`
      : `${theme.search.passive}`
```
and then use it like `border-bottom: ${variantColor};`

Reply:
Yeah good idea. I'll get that done. Thanks.